ops/services/alerts/app_service_metrics/exceptions.tf

```diff
@@ -4,15 +4,78 @@ resource "azurerm_monitor_diagnostic_setting" "collect_appserviceconsolelogs" {
   target_resource_id         = var.app_service_id
   log_analytics_workspace_id = data.azurerm_log_analytics_workspace.global.id
 
+  log {
+    category = "AppServiceAntivirusScanAuditLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
   log {
     category = "AppServiceConsoleLogs"
     enabled  = true
-
     retention_policy {
       days    = 7
       enabled = true
     }
   }
+  log {
+    category = "AppServiceFileAuditLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
+  log {
+    category = "AppServiceHTTPLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
+  log {
+    category = "AppServiceIPSecAuditLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
+  log {
+    category = "AppServicePlatformLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
+  log {
+    category = "AppServiceAppLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
+  log {
+    category = "AppServiceAuditLogs"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
+  metric {
+    category = "AllMetrics"
+    enabled  = false
+    retention_policy {
+      days    = 0
+      enabled = false
+    }
+  }
```

Comment:
Lots of disabled logs here, curious if it was already disabled before or this is turning it off now?

Reply:
This was all previously disabled, I explain a bit more [here](https://github.com/CDCgov/prime-simplereport/pull/4840#discussion_r1091436479).